### ISSUE TITLE
Safari: Wrong focus indicator on Input

### ISSUE THREAD

**Issue by jurokapsiar** [microsoft/fluent-ui-react#2376]
## Bug Report
Safari adds a weird focus indicator on Input and Dropdown (searchable):

![image](https://user-images.githubusercontent.com/13742664/74953827-78b4bd00-5402-11ea-9b2f-f67119e8996f.png)
![image](https://user-images.githubusercontent.com/13742664/74953851-82d6bb80-5402-11ea-963a-d2eae4442432.png)

### Expected Result
Focus indicator should be defined by theme

### Actual Result
Safari adds a blue ring
